packages/client/hmi-client/src/views/Document.vue

```diff
@@ -1,91 +1,92 @@
 <template>
 	<section class="doc-view-container">
 		<div v-if="doc">
+			<div class="journal">{{ doc.journal }}</div>
 			<div v-if="docLink" class="title">
 				<a :href="docLink" target="_blank" rel="noreferrer noopener">{{ doc.title }}</a>
```

Comment:
We should remove `target="_blank"`. It's up to the user to decide how to open a link.